Makefile

```diff
@@ -4,8 +4,8 @@ GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 GIT_COMMIT_ID ?= $(shell git rev-parse --short HEAD)
 
 # Image URL to use all building/pushing image targets
-REG ?= ghcr.io
```

Comment:
Let's not modify this.

Reply:
it's a wrong commit... will fix this with rebase